docs/components/board/upboard.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Configure an upboard board"
+linkTitle: "upboard"
+weight: 70
+type: "docs"
+description: "Configure an upboard board."
+images: ["/icons/components/board.svg"]
+tags: ["board", "components"]
+# SMEs: Susmita
+---
+
+Configure an `upboard` board to integrate an Intel-based board like the [UP4000](https://github.com/up-board/up-community/wiki/Pinout_UP4000) into your robot.
+
+## Set up requirements
+
+Flash your Intel-based board with:
+
+1. The Ubuntu `"Server install image"` version of the operating system. For example, `ubuntu-22.04.2-live-server-amd64.iso`.
+Follow [these instructions](https://ubuntu.com/tutorials/install-ubuntu-server) to do so.
+1. The [pin control driver](https://github.com/up-division/pinctrl-upboard).
+Follow [these instructions](https://github.com/up-division/pinctrl-upboard) to do so.
+This driver stabilizes the [GPIO pin mapping definition](https://github.com/up-board/up-community/wiki/Pinout) on the board to make it identical to a [`pi`](/components/board/pi/) board.
+
```

Comment:
Should this include that they need to install viam-server and connect it to the app? Not that they can't technically configure it before it's connected, but I think we should point them in the right direction